systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -13,7 +13,8 @@
     comtraj = [];
     mu = 0.5;
     plan_shift = zeros(6,1);
-    plan_shift_z_only = false;
+    plan_shift_zmp_inds = 1:2;
+    plan_shift_body_motion_inds = 3;
```

Comment:
what would happen if these two were not mutually exclusive?  would something break?